test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -2441,6 +2442,10 @@ protected static RestClient createRestClient(
      */
     protected void setupSuiteScopeCluster() throws Exception {}
 
+    protected boolean autoManageVotingExclusions() {
```

Comment:
Could we add a comment indicating that this is a workaround that we should remove once https://github.com/elastic/elasticsearch/issues/98055 is resolved?